go/tools/builders/stdlib.go

```diff
@@ -56,6 +56,13 @@ func run(args []string) error {
 	// TODO(#1357): also take absolute paths of includes and other paths in flags.
 	os.Setenv("CC", abs(os.Getenv("CC")))
 
+	// Ensure paths are absolute.
+	absPaths := []string{}
+	for _, path := range strings.Split(os.Getenv("PATH"), ":") {
```

Comment:
Looks good except `os.PathListSeparator` instead of `":"` for splitting and joining. The separator is `";"` on Windows.

Reply:
Awesome! Done.

Reply:
Sorry, now done for real. I found `filepath.SplitList` to get the split.